### ISSUE TITLE
Merge the solution branch

### ISSUE THREAD

**Issue by oliverjam** [foundersandcoders/ws-database-testing#36]
I think most FAC workshops have moved away from a separate `solutions` branch to just putting a folder of solutions on `master`. That would avoid the problem this repo has now, where `solutions` is a fair bit behind `master` and will inevitably not stay in sync.

**Comment by oliverjam** [comment]
Hmm I'm now struggling to decide how to do it. Should the repo be split into two `workshop` and `solution` directories with their own `package.json`s? Since part of the solution involves adding an `npm script`

**Comment by mattlub** [comment]
is it ok to just have the files which have changed in the solutions folder? i.e. as a reference, but not to actually run

**Comment by oliverjam** [comment]
Hmm that might work. It'll end up pretty similar to have a separate branch though, although less likely to get forgotten about

**Comment by mattlub** [comment]
true. I guess it's hard to maintain if you can't check it actually runs. I guess this...

> Should the repo be split into two workshop and solution directories with their own package.jsons?

is not such a terrible thing.

Your choice 😄
